data_portal/models/fastqlistrow.py

```diff
@@ -28,6 +29,20 @@ def get_by_keyword(self, **kwargs) -> QuerySet:
             qs = qs.filter(rglb__in=qs_meta)
             kwargs.pop('project_owner')
 
+        project_name = kwargs.get('project_name', None)
+        if project_name:
+            q = self.reduce_multi_values_qor('project_name', project_name)
+            qs_meta = LabMetadata.objects.filter(q).values("library_id")
+            qs = qs.filter(rglb__in=qs_meta)
+            kwargs.pop('project_name')
+
+        type_ = kwargs.get('type', None)
+        if type_:
+            q = self.reduce_multi_values_qor('type', type_)
+            qs_lims = LIMSRow.objects.filter(q).values("library_id")
```

Comment:
Ah! Let us use `type` field from `LabMetadata` here as well. 

Not your fault! Don't know why I said Type field is only avail in LIMS, initially! (Do'h) 😅 Corrected [the issue description](https://github.com/umccr/data-portal-apis/issues/444).

Reply:
Yup! Updated to use metadata table instead. 🙂

Reply:
Thank you!